io/shared/src/main/scala/fs2/io/file2/PathApi.scala

```diff
@@ -23,7 +23,7 @@ package fs2
 package io
 package file2
 
-private[file2] trait PathApi {
+private[file2] trait PathApi extends Any {
```

Comment:
Why universal trait?

Reply:
So that JS Path can be an `AnyVal` for `String`. But now I realize can't do this if we also want to store the filesystem in the path.